infrastructure/modules/service-bus/main.tf

```diff
@@ -27,12 +27,27 @@ resource "azurerm_servicebus_queue" "doc_index_updater_service_bus_queue" {
   resource_group_name = var.resource_group_name
 }
 
+resource "azurerm_servicebus_queue_authorization_rule" "doc_index_updater_service_bus_queue_auth_rule" {
+  name = "${local.name}-${local.queue_names[count.index]}-auth"
+
+
+  count               = length(local.queue_names)
+  namespace_name      = azurerm_servicebus_namespace.doc_index_updater_service_bus.name
+  queue_name          = "${local.name}-${local.queue_names[count.index]}-queue"
+  resource_group_name = var.resource_group_name
+
+  listen = true
+  manage = true
+  send   = true
+}
+
+
 
 resource "azurerm_redis_cache" "doc_index_updater_redis" {
   name = var.name
 
   capacity            = 0
-  enable_non_ssl_port = true # FIXME: Set to true to enable development / Change this before go to prod
+  enable_non_ssl_port = false # FIXME: Set to true to enable development / Change this before go to prod
```

Comment:
I think we can get rid of the `FIXME` here.

Reply:
done